osafw-app/App_Code/models/DBUpdates.cs

```diff
@@ -0,0 +1,65 @@
+// DBUpdates model class
+//
+// Part of ASP.NET osa framework  www.osalabs.com/osafw/asp.net
+// (c) 2009-2023 Oleg Savchuk www.osalabs.com
+
+using System.Collections;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+
+namespace osafw;
+
+public class DBUpdates : FwModel
+{
+    public const int STATUS_APPLIED = 50;
+    public DBUpdates() : base()
```

Comment:
why not inherit from FwModel? you will then have all the standard methods to work with a table